src/08-accessing.js

```diff
@@ -171,7 +194,12 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
+  const arr = Object.assign(person.names, person.address);
```

Comment:
We should probably choose a better variable name here. Remember this variable represents an `Object`, not an `Array`. Also, I would have not chosen to use `Object.assign`. Remember this method _mutates_ the first argument passed to it. Now in memory the value of person.names has been manipulated to contain the data in person.address. Remember that this problem called for you to create a _new_ `Object` in memory, not manipulate the input.